migrations/826-user-drop-source.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE users DROP COLUMN `source`;
```

Comment:
Yeah, I'm torn between not liking having a dangling column and not liking throwing away data that might be useful. But the more I think about it, the more I think it doesn't really matter, so let's just drop it.